Gemfile

```diff
@@ -45,7 +45,7 @@ gem 'pundit'
 
 # Address Verification & Geocoding
 gem 'mainstreet'
-gem 'geocoder', github: 'ankane/geocoder', branch: 'smarty_streets_international'
```

Comment:
We had sourced geocoder from github because this branch was required by `mainstreet` at the time. The branch has since been merged to master and released in 1.5.1.

https://github.com/ankane/mainstreet/commit/959fbe2e712cc9daa4e5bab714d681c3ce685a27#diff-04c6e90faac2675aa89e2176d2eec7d8